### PR TITLE
fixed QGroupBox checkboxes and other minor bugs

### PR DESCRIPTION
**Pull request by pgilfernandez**
- Updated to v2.21
- minor fixes
- typos
- added QGroupBox checkboxes styles

Cheers

### CODE REVIEW

File: src/Gui/Stylesheets/Dark-green.qss [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /*
 ABOUT
 ============================================================================================================
-version 2.20
-Qt theme (stylesheet) specially developed for FreeCAD (https://www.freecadweb.org/).
+version 2.21
+Qt theme (stylesheet) specially developed for FreeCAD (http://www.freecadweb.org/).
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Dark-green.qss]: why change this? It was correct. Please revert back to `https` please

File: src/Gui/Stylesheets/Dark-blue.qss [hunk 2]
```diff
@@ -57,7 +57,8 @@ If you would like to change the overall look/style of the theme, just find and r
 
 KNOWN BUGS and TO DO
 ============================================================================================================
-- please, follow the link to get updated information: https://forum.freecadweb.org/viewtopic.php?f=10&t=12417
+Please, follow the link to get updated information: http://forum.freecadweb.org/viewtopic.php?f=10&t=12417
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Dark-blue.qss]: Please change to `https://`  
Thanks.

File: src/Gui/Stylesheets/Dark-blue.qss [hunk 3]
```diff
@@ -1,8 +1,8 @@
 /*
 ABOUT
 ============================================================================================================
-version 2.20
-Qt theme (stylesheet) specially developed for FreeCAD (https://www.freecadweb.org/).
+version 2.21
+Qt theme (stylesheet) specially developed for FreeCAD (http://www.freecadweb.org/).
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Dark-blue.qss]: why change this? It was correct. Please revert back to `https` please

File: src/Gui/Stylesheets/Dark-green.qss [hunk 4]
```diff
@@ -57,7 +57,8 @@ If you would like to change the overall look/style of the theme, just find and r
 
 KNOWN BUGS and TO DO
 ============================================================================================================
-- please, follow the link to get updated information: https://forum.freecadweb.org/viewtopic.php?f=10&t=12417
+Please, follow the link to get updated information: http://forum.freecadweb.org/viewtopic.php?f=10&t=12417
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Dark-green.qss]: `https://`

File: src/Gui/Stylesheets/Dark-orange.qss [hunk 5]
```diff
@@ -1,8 +1,8 @@
 /*
 ABOUT
 ============================================================================================================
-version 2.20
-Qt theme (stylesheet) specially developed for FreeCAD (https://www.freecadweb.org/).
+version 2.21
+Qt theme (stylesheet) specially developed for FreeCAD (http://www.freecadweb.org/).
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Dark-orange.qss]: Please revert back to `https` please

File: src/Gui/Stylesheets/Dark-orange.qss [hunk 6]
```diff
@@ -57,7 +57,8 @@ If you would like to change the overall look/style of the theme, just find and r
 
 KNOWN BUGS and TO DO
 ============================================================================================================
-- please, follow the link to get updated information: https://forum.freecadweb.org/viewtopic.php?f=10&t=12417
+Please, follow the link to get updated information: http://forum.freecadweb.org/viewtopic.php?f=10&t=12417
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Dark-orange.qss]: `https://`

File: src/Gui/Stylesheets/Darker-blue.qss [hunk 7]
```diff
@@ -1,8 +1,8 @@
 /*
 ABOUT
 ============================================================================================================
-version 2.20
-Qt theme (stylesheet) specially developed for FreeCAD (https://www.freecadweb.org/).
+version 2.21
+Qt theme (stylesheet) specially developed for FreeCAD (http://www.freecadweb.org/).
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Darker-blue.qss]: `https://`

File: src/Gui/Stylesheets/Darker-blue.qss [hunk 8]
```diff
@@ -57,7 +57,8 @@ If you would like to change the overall look/style of the theme, just find and r
 
 KNOWN BUGS and TO DO
 ============================================================================================================
-- please, follow the link to get updated information: https://forum.freecadweb.org/viewtopic.php?f=10&t=12417
+Please, follow the link to get updated information: http://forum.freecadweb.org/viewtopic.php?f=10&t=12417
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Darker-blue.qss]: `https://`

File: src/Gui/Stylesheets/Darker-green.qss [hunk 9]
```diff
@@ -1,8 +1,8 @@
 /*
 ABOUT
 ============================================================================================================
-version 2.20
-Qt theme (stylesheet) specially developed for FreeCAD (https://www.freecadweb.org/).
+version 2.21
+Qt theme (stylesheet) specially developed for FreeCAD (http://www.freecadweb.org/).
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Darker-green.qss]: `https://`

File: src/Gui/Stylesheets/Darker-green.qss [hunk 10]
```diff
@@ -57,7 +57,8 @@ If you would like to change the overall look/style of the theme, just find and r
 
 KNOWN BUGS and TO DO
 ============================================================================================================
-- please, follow the link to get updated information: https://forum.freecadweb.org/viewtopic.php?f=10&t=12417
+Please, follow the link to get updated information: http://forum.freecadweb.org/viewtopic.php?f=10&t=12417
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Darker-green.qss]: `https://`

File: src/Gui/Stylesheets/Darker-orange.qss [hunk 11]
```diff
@@ -1,8 +1,8 @@
 /*
 ABOUT
 ============================================================================================================
-version 2.20
-Qt theme (stylesheet) specially developed for FreeCAD (https://www.freecadweb.org/).
+version 2.21
+Qt theme (stylesheet) specially developed for FreeCAD (http://www.freecadweb.org/).
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Darker-orange.qss]: `https://`

File: src/Gui/Stylesheets/Darker-orange.qss [hunk 12]
```diff
@@ -57,7 +57,8 @@ If you would like to change the overall look/style of the theme, just find and r
 
 KNOWN BUGS and TO DO
 ============================================================================================================
-- please, follow the link to get updated information: http://forum.freecadweb.org/viewtopic.php?f=10&t=12417
+Please, follow the link to get updated information: http://forum.freecadweb.org/viewtopic.php?f=10&t=12417
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Darker-orange.qss]: `https://`

File: src/Gui/Stylesheets/Light-blue.qss [hunk 13]
```diff
@@ -1,8 +1,8 @@
 /*
 ABOUT
 ============================================================================================================
-version 2.20
-Qt theme (stylesheet) specially developed for FreeCAD (https://www.freecadweb.org/).
+version 2.21
+Qt theme (stylesheet) specially developed for FreeCAD (http://www.freecadweb.org/).
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Light-blue.qss]: `https://`

File: src/Gui/Stylesheets/Light-blue.qss [hunk 14]
```diff
@@ -57,7 +57,8 @@ If you would like to change the overall look/style of the theme, just find and r
 
 KNOWN BUGS and TO DO
 ============================================================================================================
-- please, follow the link to get updated information: https://forum.freecadweb.org/viewtopic.php?f=10&t=12417
+Please, follow the link to get updated information: http://forum.freecadweb.org/viewtopic.php?f=10&t=12417
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Light-blue.qss]: `https://`

File: src/Gui/Stylesheets/Light-green.qss [hunk 15]
```diff
@@ -1,8 +1,8 @@
 /*
 ABOUT
 ============================================================================================================
-version 2.20
-Qt theme (stylesheet) specially developed for FreeCAD (https://www.freecadweb.org/).
+version 2.21
+Qt theme (stylesheet) specially developed for FreeCAD (http://www.freecadweb.org/).
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Light-green.qss]: `https://`

File: src/Gui/Stylesheets/Light-green.qss [hunk 16]
```diff
@@ -57,7 +57,8 @@ If you would like to change the overall look/style of the theme, just find and r
 
 KNOWN BUGS and TO DO
 ============================================================================================================
-- please, follow the link to get updated information: https://forum.freecadweb.org/viewtopic.php?f=10&t=12417
+Please, follow the link to get updated information: http://forum.freecadweb.org/viewtopic.php?f=10&t=12417
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Light-green.qss]: `https://`

File: src/Gui/Stylesheets/Light-orange.qss [hunk 17]
```diff
@@ -1,8 +1,8 @@
 /*
 ABOUT
 ============================================================================================================
-version 2.20
-Qt theme (stylesheet) specially developed for FreeCAD (https://www.freecadweb.org/).
+version 2.21
+Qt theme (stylesheet) specially developed for FreeCAD (http://www.freecadweb.org/).
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Light-orange.qss]: `https://`

File: src/Gui/Stylesheets/Light-orange.qss [hunk 18]
```diff
@@ -57,7 +57,8 @@ If you would like to change the overall look/style of the theme, just find and r
 
 KNOWN BUGS and TO DO
 ============================================================================================================
-- please, follow the link to get updated information: https://forum.freecadweb.org/viewtopic.php?f=10&t=12417
+Please, follow the link to get updated information: http://forum.freecadweb.org/viewtopic.php?f=10&t=12417
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Light-orange.qss]: `https://`

**Comment by pgilfernandez** [comment]
You are right, let me add a new pull request to re-fix it...

Sorry for the troubles

**Comment by pgilfernandez** [comment]
done,

This [PR](https://github.com/FreeCAD/FreeCAD/pull/2888) should fix it

**Pull request merged**